starter-code/french-onion-soup-starter-code/css/main.css

```diff
@@ -7,3 +7,64 @@ html {
   font-family: 'Open Sans', sans-serif;
   font-size: 18px
 }
+
+header{
```

Comment:
If you added a header to your html you would be able to put all your css in one file since both files are almost the same.

```
header .soup {
background-image: url("../images/soup.jpg");
}
```